micro_dl/utils/train_utils.py

```diff
@@ -115,6 +116,7 @@ def split_train_val_test(sample_set, train_ratio, test_ratio,
     num_test = int(test_ratio * num_samples)
     num_test = max(num_test, 1)
 
+    np.random.seed(random_seed)
```

Comment:
I think the seed needs to be reset for the val split as well?

Reply:
Hmm... not sure about that! The function is running in a single thread and I think that the state of the random number generator will not be reset between the calls for test and val. Tried it for two sep python sessions and the returned samples were the same! Please let me know if you see any different behaviour!

Reply:
Yes you're right! I got this result when testing the behavior of np.random.choice and got confused:
`np.random.choice(10,3)
Out[7]: array([5, 0, 3])
np.random.choice(10,3)
Out[8]: array([3, 7, 9])
np.random.seed(0)
np.random.choice(10,3)
Out[10]: array([5, 0, 3])
np.random.choice(10,3)
Out[11]: array([3, 7, 9])`

The second call of np.random.choice returns different numbers from the first call when the seed is not reset, but the numbers are the same as the the previous second call after resetting the seed, so in this case we don't need to reset the seed 

Sorry for the confusion!